### PR TITLE
Fix passcode accessibility.

### PR DESCRIPTION
**Pull request by brandonpage**

### CODE REVIEW

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m [hunk 1]
```diff
@@ -318,12 +335,14 @@ - (void)validatePasscodeConfirmed:(NSString *)validPasscode
 {
     [self resetReaminingAttemps];
     [self.verifyDelegate passcodeVerified];
+    UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, [SFSDKResourceUtils localizedString:@"accessibilityUnlockAnnouncement"]);
```
Comment (trooper2013) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m]: Should we do this only if UIAccessibilityIsVoiceOverRunning()

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m [hunk 2]
```diff
@@ -318,12 +335,14 @@ - (void)validatePasscodeConfirmed:(NSString *)validPasscode
...
 }
 
 - (void)validatePasscodeFailed
 {
     [self resetReaminingAttemps];
     [self.verifyDelegate passcodeFailed];
+    UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, [SFSDKResourceUtils localizedString:@"accessibilityLoggedOutAnnouncement"]);
```
Comment (trooper2013) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m]: Should we do this only if UIAccessibilityIsVoiceOverRunning()

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m [hunk 3]
```diff
@@ -118,6 +118,12 @@ - (void)loadView
     self.passcodeTextView.delegate = self;
     self.passcodeTextView.layer.borderWidth = kSFViewBoarderWidth;
     self.passcodeTextView.accessibilityIdentifier = @"passcodeTextField";
+    self.passcodeTextView.accessibilityLabel = [SFSDKResourceUtils localizedString:@"accessibilityPasscodeFieldLabel"];
+    self.passcodeTextView.secureTextEntry = YES;
+    self.passcodeTextView.isAccessibilityElement = YES;
+    if (self.passcodeLengthKnown) {
+        self.passcodeTextView.accessibilityHint = [[NSString alloc] initWithFormat:[SFSDKResourceUtils localizedString:@"accessibilityPasscodeLengthHint"], self.viewConfig.passcodeLength];
```
Comment (trooper2013) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m]: Can you switch this to ```[NSString stringWithFormat:]``` instead?

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeCreateController.m [hunk 4]
```diff
@@ -207,7 +216,9 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
                 [self.passcodeTextView clearPasscode];
                 [self.passcodeTextView refreshView];
                 [self.passcodeInstructionsLabel setText:[SFSDKResourceUtils localizedString:@"passcodesDoNotMatchError"]];
+                UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, self.passcodeInstructionsLabel.text);
```
Comment (trooper2013) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeCreateController.m]: Should we check for UIAccessibilityIsVoiceOverRunning()?

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeCreateController.m [hunk 5]
```diff
@@ -218,14 +229,15 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
             //Change labels for confirm passcode
             [self.navigationItem setTitle:[SFSDKResourceUtils localizedString:@"verifyPasscodeNavTitle"]];
             [self.passcodeInstructionsLabel setText:[SFSDKResourceUtils localizedString:@"passcodeConfirmInstructions"]];
+            UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, self.passcodeInstructionsLabel.text);
```
Comment (trooper2013) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeCreateController.m]: Should we check for  UIAccessibilityIsVoiceOverRunning()

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m (hunk 1 above)
Comment (brandonpage) [reply libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKPasscodeVerifyController.m]: We could.  But it should do nothing if voice over is off, right?

**Pull request merged**